docs/pages/technology/vc-concepts.rst

```diff
@@ -0,0 +1,103 @@
+.. _vc-concepts:
+
+VC Concept mapping
+##################
+
+Verifiable Credentials are, by nature, very dynamic. This is reflected by the data format: either JSON or fields within a JWT (basically also JSON).
+There are two ways to cope with this: 1) transform every credential to a fixed concept. This creates a reliable DB but requires custom code for each credential type.
+Or 2) store the credential in a document store and provide a way to interact with the credentials through some interface where magic happens.
+
+We opted for the latter and call it concept mappings. A *concept* represents a selection of data that is needed to support bolts.
+An example concept is a care organization. A key part of a care organization is it's name. This name is given to a DID via a Verifiable Credential.
+Multiple credentials can give a DID its name, the concept mappings makes sure only a single interface is required at the side of the Nuts node.
+
+See :ref:`preconfigured concepts <default-concepts>` for a list of supported concepts.
+
+templates
+*********
+
+Credentials are mapped on a concept through templates.
+
+An example concept mapping template looks like this:
+
+.. code-block:: json
+
+    {
+        "id": "<<id>>",
+        "issuer": "<<issuer>>",
+        "type": "ExampleCredential@{1_1},{2_1}",
+        "credentialSubject": {
+            "id": "<<subject>>@{2_2}",
+            "company": {
+                "name": "<<company.name>>@{1_2}",
+                "city": "<<company.city>>"
+            }
+        }
+    }
+
+it follows the syntax of a Verifiable Credential. Only the parts of the credential that are mapped or have a fixed value are in the template.
+The example template above introduces the `company` concept. That name must also be used in the APIs.
+A concept is automatically derived from any *conceptValue* with a `.` in it. Any field that is encapsulated between `<<` and `>>` is called a *conceptValue*.
+This *conceptValue* is also used in the APIs and CLI. It can be used as JSON like:
+
+.. code-block:: json
+
+    {
+        "company": {
+            "name": "X",
+            "city": "Y"
+        }
+    }
+
+or as key-value:
+
+.. code-block:: bash
+
+    company.name=X
+    company.city=Y
+
+generic fields
```

Comment:
I think we call 'm "defaults" or something in code, isntead of "generic"? Let's decide on 1 term

Reply:
both are valid to use. Generic refers to fields common to all VCs and defaults refer to the values.